components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/cache/AuthorizationGrantCacheEntry.java

```diff
@@ -292,4 +294,24 @@ public String getAuthorizationCode() {
 
         return authorizationCode;
     }
+
+
+    /**
+     *
+     * @return  Whether the flow has request object or not.
+     */
+    public boolean isContainsRequestObject() {
```

Comment:
what if we change this as `isContainsRequestObject` - > `isRequestObjectFlow` or something else